src/Functions/Public/catalog-service/Get-vRARequestResult.ps1

```diff
@@ -0,0 +1,94 @@
+
+function Get-vRARequestResult {
```

Comment:
Possible function rename? I feel that Get-vRARequestDetail would suit better as Get-vRARequest actually returns the result of the request.. @jonathanmedd Interested in your views